03week/ticTacToe.js

```diff
@@ -24,39 +32,189 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  // Your code
+  if(board[0][0] == 'X' && board[0][1] == 'X' && board[0][2] == 'X') {
+    playerTurn = 'X';
+    console.log(playerTurn + ' Wins!!!' + '\n');
+    return true;
+  } else if(board[0][0] == 'O' && board[0][1] == 'O' && board[0][2] == 'O') {
+    playerTurn = 'O';
+    console.log(playerTurn + ' Wins!!!' + '\n');
+    return true;
+  } else if(board[1][0] == 'X' && board[1][1] == 'X' && board[1][2] == 'X') {
+    playerTurn = 'X';
+    console.log(playerTurn + ' Wins!!!' + '\n');
+    return true;
+  } else if(board[1][0] == 'O' && board[1][1] == 'O' && board[1][2] == 'O') {
+    playerTurn = 'O';
+    console.log(playerTurn + ' Wins!!!' + '\n');
+    return true;
+  } else if(board[2][0] == 'X' && board[2][1] == 'X' && board[2][2] == 'X') {
+    playerTurn = 'X';
+    console.log(playerTurn + ' Wins!!!' + '\n');
+    return true;
+  } else if(board[2][0] == 'O' && board[2][1] == 'O' && board[2][2] == 'O') {
+    playerTurn = 'O';
+    console.log(playerTurn + ' Wins!!!' + '\n');
+    return true;
+  }
```

Comment:
this works, but it is VERY long. you shouldn't check if the position is equal to "X" or "O", you should check against playerTurn. This will cut the code in half.